package.json

```diff
@@ -145,6 +149,10 @@
                 {
                     "command": "extension.vsKubernetesRefreshExplorer",
                     "when": "view == extension.vsKubernetesExplorer"
+                },
+                {
+                    "command": "extension.vsKubernetesUseContext",
+                    "when": "view == extension.vsKubernetesExplorer"
```

Comment:
We seem to have two `when` clauses for the `extension.vsKubernetesUseContext` command: is this intentional?

Reply:
YES, the second `when` is intentional. It's used to restrict this command only to be visible at the tree explorer context menu, not listed at the drop down command palette via `Ctrl + Shift + P`.

Reply:
Ooh, that's really handy.  I need to apply this to some of my existing commands I think!